docs/docs/api-reference/utils/waitForAny.md

```diff
@@ -3,4 +3,21 @@ title: waitForAny(dependencies)
 sidebar_label: waitForAny()
 ---
 
-TODO Documentation
+A concurrency helper that returns a set of [`Loadables`](/docs/api-reference/core/Loadable) for the current state of the requested dependencies.  It waits until at least one of the dependencies are available.
```

Comment:
`Loadable` should be singular-- we can follow it with an "s": `Loadable`s

Reply:
nit grammar: use "is" vs "are":

"It waits until at least one of the dependencies is available"